### PR TITLE
Document keyboard shortcuts

### PR DESCRIPTION
**Pull request by adamlazik1**
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

**Review by MariaAga — APPROVED** [review]

File: guides/common/modules/con_using-keyboard-shortcuts.adoc [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[id="using-keyboard-shortcuts_{context}"]
+= Using Keyboard Shortcuts
+
+You can use keyboard shortcuts to quickly focus search bars.
+
+To focus the vertical navigation search bar, press `CTRL + SHIFT + F`.
+
+To focus the page search bar, press `/`.
```
Comment (maximiliankolb) [inline guides/common/modules/con_using-keyboard-shortcuts.adoc:8]: Can you please try https://docs.asciidoctor.org/asciidoc/latest/macros/keyboard-macro/?

Also, maybe make line 6+8 an unordered list?

Comment (adamlazik1) [reply guides/common/modules/con_using-keyboard-shortcuts.adoc]: Seems like the macro does not work:
image
so I am reverting that.

Comment (maximiliankolb) [reply guides/common/modules/con_using-keyboard-shortcuts.adoc]: Ah, I remember that they need ":experimental:" set. Probably not something we want to run in prod right now. See https://docs.asciidoctor.org/asciidoc/latest/macros/keyboard-macro/

Thanks for testing anyway.

**Review by mjivraja — APPROVED** [review]

File: guides/common/modules/con_using-keyboard-shortcuts.adoc (hunk 1 above)
Comment (adamlazik1) [reply guides/common/modules/con_using-keyboard-shortcuts.adoc]: Hmm, I actually did not notice that. To be honest, I wouldn't mind having the macro in the module, it does say that the macro is stable. Shall I add it back?

Comment (mjivraja) [reply guides/common/modules/con_using-keyboard-shortcuts.adoc]: There is a catch. They have also mentioned that they might change the name of attribute (:experimental:) after their internal decision. We can delay adding macro?

Comment (maximiliankolb) [reply guides/common/modules/con_using-keyboard-shortcuts.adoc]: If it's not cherry-picked, then trying it out is really OK for me. Let's see what happens.

Comment (adamlazik1) [reply guides/common/modules/con_using-keyboard-shortcuts.adoc]: That as fair enough. I will merge as-is after QE review.

**Comment by adamlazik1** [comment]
Merged and cherry-picked:
   6ab251459..7e290701b  3.9 -> 3.9

**Pull request merged**